ansible/roles/tomcat/tasks/main.yml

```diff
@@ -98,15 +98,15 @@
   notify:
   - restart tomcat
 
-- name: enable the HTTP 1.1 connector (Tomcat-9)
```

Comment:
This section `enable the HTTP 1.1 connector (Tomcat-9)` is still disabled. Please revert this change. Make it conditional to exec_jar if requried.

Reply:
I was unaware of any new issue with this role as I have recently used it on new Ubuntu 22 servers without error. While still avoiding fixing the problem, a less severe workaround is to use `ignore_errors: yes`.

Reply:
The task fails with the error /usr/bin/env: ‘python3’: No such file or directory, despite python3 being installed and accessible. We can successfully run python3 from the terminal when logged in as the tomcat user.

Reply:
If I understand correctly, this is running on an older server, so I expect it might well be only specific to that one server since it does run without error in other environments. It is very odd that the other tomcat9_connector tasks run without this same error. 

My intention is to not touch something that is working, and this task is definitely working in some environments. 

While it is not ideal, `ignore_errors: yes` can be used if the environment where it is not working has that connector already configured in tomcat.

Reply:
I have added 'ignore-error:yes'